_sass/main.scss

```diff
@@ -6,4 +6,22 @@
 // Run the vanilla framework
 @include vanilla;
 
-@import 'base';
+@import
+
+'base',
+'box',
+'lists',
+'footer',
+'hero',
+'media-unit';
+
+@media only screen and (min-width: $breakpoint-medium) {
+
+  .flex-wrap {
+    display: flex;
+
+    > .four-col {
+      display: flex;
```

Comment:
Can you explain this? Wont this disturb the grid? Do you mean to nest flex element?